pipeline-scripts/buildlib.groovy

```diff
@@ -306,4 +341,79 @@ def with_virtualenv(path, f) {
     return withEnv(env, f)
 }
 
-return this
+// Parse record.log from OIT into a map
```

Comment:
What does the map look like (I know I can just read the code, but save someone time)?